apps/src/templates/sectionAssessments/MatchAssessmentsOverviewTable.story.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {UnconnectedMatchAssessmentsOverviewTable} from './MatchAssessmentsOverviewTable';
+import {
+  matchQuestionWith2Pairs,
+  matchQuestionWith4Pairs
+} from './assessmentsTestHelpers';
+
+export default storybook => {
+  return storybook
+    .storiesOf('SectionAssessments/MatchAssessmentsOverviewTable', module)
+    .addStoryTable([
+      {
+        name: 'Assessment match with 4 option/answer pairs',
+        description: 'Ability to see assessment overview for a section',
+        story: () => (
+          <UnconnectedMatchAssessmentsOverviewTable
+            questionAnswerData={matchQuestionWith4Pairs}
+            openDialog={() => {}}
+            setQuestionIndex={() => {}}
+          />
+        )
+      },
+      {
+        name: 'Assessment match with 2 option/answer pairs',
+        description: 'Ability to see assessment overview for a section',
+        story: () => (
+          <UnconnectedMatchAssessmentsOverviewTable
+            questionAnswerData={matchQuestionWith2Pairs}
+            openDialog={() => {}}
+            setQuestionIndex={() => {}}
+          />
+        )
+      }
+    ]);
+};
```

Comment:
yay! can you make a note to check this when Storybook is back up?

Reply:
storybook can be used by patching in https://github.com/code-dot-org/code-dot-org/tree/upgrade-storybook and then running `yarn; yarn build ; yarn storybook`

Reply:
checked!